lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -60,11 +60,16 @@
   }
 
   .tagline {
+    width: 75%;
+    margin: auto;
     font-size: $font-small;
+    color: $dark-gray;
+    font-style: italic;
   }
 
   .form-actions {
     padding-top: gutter();
+    margin-bottom: 24px;
```

Comment:
there's a sass variable for this... I think it's `$base-spacing` or one of the other ones.